compiler/ir/backend.py/src/org/jetbrains/kotlin/ir/backend/py/transformers/irToPy/PyIntrinsicTransformers.kt

```diff
@@ -314,11 +251,6 @@ class PyIntrinsicTransformers(backendContext: JsIrBackendContext) {
                 val array = args[0]
                 val index = args[1]
                 val value = args[2]
-//                Assign(  // todo: this is a statement but an expression is required, so using __setitem__ call below for now
-//                    targets = listOf(Subscript(value = array, slice = index, ctx = Store)),
-//                    value = value,
-//                    type_comment = null,
-//                )
                 Call(
```

Comment:
Let's save the todo here.
```suggestion
                // todo: this is a statement but an expression is required, so using __setitem__ call below for now
                Call(
```